sdk/cosmos/examples/document_00.rs

```diff
@@ -79,11 +80,7 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
         Some(db) => db,
         None => {
             database_client
-                .create_database(
-                    azure_core::Context::new(),
-                    DATABASE,
-                    azure_cosmos::operations::create_database::Options::new(),
-                )
+                .create_database(Context::new(), DATABASE, create_database::Options::new())
```

Comment:
Is this idiomatic to call `new()`, or should we implement the `Default` trait? I see that used in quite a few samples, though they are pretty basic.

Reply:
Implementing default is probably a good idea for this struct. Other options structs won't have defaults since they might have required options. Creating a collection, for example, requires a partition key (at least for now).